packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -51,6 +51,39 @@ String mainAxisAssetImageUrl(MainAxisAlignment alignment) {
   return 'assets/img/story_of_layout/main_axis_alignment/${describeEnum(alignment)}.png';
 }
 
+List<double> computeRenderSizes({
+  @required Iterable<double> sizes,
+  @required double smallestSize,
+  @required double largestSize,
+  @required double smallestRenderSize,
+  @required double largestRenderSize,
+  @required double maxSize,
+  bool shouldForceToOccupyMaxSize = true,
+}) {
+  final transformToRenderSize = (largestRenderSize) => [
+        for (var s in sizes)
```

Comment:
Something that I discovered I liked from using Go was to use short, math-like variable names when I was doing complex algebra in my code.  I may be the only person on Flutter with this preference, but consider aliasing the variables to local names the same as we used on the whiteboard:

```dart
computeRenderSizes({...}) {
  double ss, ls, srs, lrs, lrs;
  // assign from the parameters because callers of this method shouldn't be expected to know what a two-letter name means.
  List<int> transformToRenderSize(double ls) => [
    for (var s in sizes)
      ((s - ss) * (lrs - srs) / (ls - ss)) + srs
  ];
  ...similar for overriding the size.
}
```

For code like this, I think that's much easier to read.  Feel free

Reply:
I agree